src/Calendar/index.d.ts

```diff
@@ -21,6 +21,7 @@ export interface CalendarProps {
   showMonthDropdown?: boolean;
   shouldCloseOnSelect?: boolean;
   locale?: string | { distanceInWords?: {}; format?: {} };
+  firstDayOfWeek?: number
```

Comment:
Please be more specific.
```suggestion
  firstDayOfWeek?: 1 | 2 | 3 | 4 | 5 | 6;
```
I think this will work

Reply:
maybe it's 0 to 5

Reply:
Fixed :) it is indeed 0-6